Hummingbird.docc/Articles/PersistentData.md

```diff
@@ -47,7 +47,9 @@ To access values in the `persist` key/value store you use
 ```swift
 let value = try await persist.get(key: "mykey", as: MyValueType.self)
 ```
-This returns he value associated with the key or `nil` if that value doesn't exist or is not of the type requested.
+
+This returns the value associated with the key or `nil` if that value doesn't exist.
+If the value is not of the expected type, this will throw ``PersistError.invalidConversion``.
```

Comment:
This should fix the validation error
```suggestion
If the value is not of the expected type, this will throw ``PersistError/invalidConversion``.
```

Reply:
This is a good call though. We should try to catch other places where errors are thrown but not documented